website/docs/core-concepts/validate/terraform-variables.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Terraform Input Variables Validation
+sidebar_position: 4
+sidebar_label: Terraform Input Variables
+description: Use OPA policies to validate Terraform input variables.
+id: terraform-variables
+---
+
+import Terminal from '@site/src/components/Terminal'
+import File from '@site/src/components/File'
+import EmbedFile from '@site/src/components/EmbedFile'
+import Intro from '@site/src/components/Intro'
+
+<Intro>
+Use [Open Policy Agent](https://www.openpolicyagent.org/docs/latest/) (OPA) policies to validate Terraform input variables.
+</Intro>
+
+## Introduction
+
+When executing `atmos terraform <sub-command>` commands, you can provide
+[Terraform input variables](https://developer.hashicorp.com/terraform/language/values/variables) on the command line
+using the `-var` flag. These variables will override the variables configured in Atmos stack manifests.
+
+For example:
+
+<Terminal>
+```shell
+atmos terraform apply <component> -s <stack> -var name=api
```

Comment:
I really don't think we should continue to support terraform CLI arguments before the double dash. It's hard to communicate what is an Atmos flag versus a terraform flag. I believe many of these single-flags don't work anymore. Also, if you run `atmos terraform --help` you'll see how much work we've put in to making this a documented experience.

Reply:
Furthermore, we now validate that the flags are supported. We error if an invalid flag is passed; for example, a typo is now an error.